acme.go

```diff
@@ -96,73 +93,7 @@ func RegisterAccount(client *lego.Client, account *Account) error {
 	return nil
 }
```

Comment:
`GetDNSProvider` and `RegisterAccount` all belong in the same file as the `AcmeClient`.